dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/objectbundle/hooks/DataSetObjectBundleHook.java

```diff
@@ -64,21 +65,28 @@
 
         if ( inputPeriods.size() > 0 )
         {
+            Date openDate, closeDate, periodStart, periodEnd;
```

Comment:
Any good reason to pre-declare these vars?

Reply:
just easier to look at. Having full getter methods in below if conditions make it difficult to read.